### DISCUSSION TITLE
Modifying "parachuting.cpp" to Apply Gravity Force to Cell in HemoCell

### DISCUSSION THREAD

**Discussion by Mansoor-Jadidi-2023** [UvaCsl/HemoCell#104] [General discussion]
Hello, HemoCell Community, 

I hope this message finds you all well. I am delighted to participate in this helpful discussion for the first time. 

My goal is to utilise the "Parachuting" example in HemoCell as a starting point for my project. However, I am eager to make some specific modifications.
Currently, my focus lies on applying the gravity force to the cell. To achieve this, I have already reoriented the tube and RBC in the "Parachuting" example, and I have disabled the Poiseuille force used to act on the fluid.
While I have made these initial changes, I am now seeking advice and suggestions from the experienced members of this community on how to proceed further. Specifically, I would greatly appreciate your insights on how to apply the gravity force to the cell while disregarding the driving force acting on the fluid.

I want to express my gratitude in advance to anyone who can spare their valuable time and knowledge to help me with this matter.
Thank you all for creating such a supportive community, and I look forward to your kind assistance.

With kind regards, 
Mansoor

**Comment by ErenC1** [comment]
Hi,

For this kind of implementation look at the Palabos forums. Because HemoCell's fluid part uses Palabos. For example:

https://palabos-forum.unige.ch/t/gravity-force/168/7
